python/source/org/libreoffice/ide/eclipse/python/JavaDebugExtraOptionsProvider.java

```diff
@@ -0,0 +1,42 @@
+package org.libreoffice.ide.eclipse.python;
+
+import org.libreoffice.ide.eclipse.core.model.config.IExtraOptionsProvider;
+import org.libreoffice.ide.eclipse.core.model.utils.SystemHelper;
+
+/**
+ * Adds extra env variables to start LibreOffice with Java setup to run in debug mode.
+ */
+public class JavaDebugExtraOptionsProvider implements IExtraOptionsProvider {
```

Comment:
Can you rename this to JavaDebugExtraOptionsProvider and investigate what is neccessary to connect the debugger to an installed LO extension?